kube_resource_report/pricing.py

```diff
@@ -144,7 +145,7 @@ def generate_ec2_price_list():
     max_price = {}
     for location in sorted(LOCATIONS.values()):
         # some regions are not available
-        if location in ("ap-northeast-3", "cn-north-1", "cn-northwest-1", "us-gov-west-1", "us-gov-east-1"):
+        if location in ("ap-northeast-3", "cn-north-1", "cn-northwest-1", "us-gov-west-1", "us-gov-east-1", "ap-east-1"):
```

Comment:
Why did you add "ap-east-1" to the list? Looks like you could get the prices for this region..

Reply:
I cannot...

```
botocore.exceptions.ClientError: An error occurred (AuthFailure) when calling the DescribeSpotPriceHistory operation: AWS was not able to validate the provided access credentials
```

You have to ask for permission for this region. You do not have access there by default.

Reply:
Yes, but EC2 prices are updated, maybe only spot is a problem

Reply:
Sorry, forget what I said :-/